### PR TITLE
Org page

### PR DESCRIPTION
**Pull request by Bit-Han**
Adding the OrgPage

### CODE REVIEW

**Review by taiwoAkindele — CHANGES_REQUESTED** [review]
Your input field behavior is different from all other input field on the project. Your placeholder should not be shifting to the border on focus. Try to negate that. Also the select box too.

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -1,112 +1,108 @@
...
-    textAlign: "center",
-    padding: "0 1rem",
-  },
-  drawerButton: {
-    display: "none",
-    "@media screen and (max-width: 900px)": {
-      display: "block"
-    }
-  }
+  // root: {
```
Comment (taiwoAkindele) [inline src/components/Navbar.tsx]: Please Remove if this is not in use

File: src/components/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Stack from '@mui/material/Stack';
+import Button  from '@mui/material/Button';
+import { height } from '@mui/system';
```
Comment (taiwoAkindele) [inline src/components/Button.tsx]: I think this is a mistake or are you using it anywhere?

Comment (taiwoAkindele) [reply src/components/Button.tsx]: Also you have conflicts on your branch

File: src/components/Navbar.tsx (hunk 1 above)
Comment (Bit-Han) [reply src/components/Navbar.tsx]: thanks i have done that

File: src/components/Button.tsx (hunk 2 above)
Comment (Bit-Han) [reply src/components/Button.tsx]: yes i have resolved the conflict thanks so much

**Review by Temitopefa — APPROVED** [review]

**Review by taiwoAkindele — APPROVED** [review]

**Pull request merged**